src/Microsoft.TestPlatform.ObjectModel/Client/Events/DiscoveryCompleteEventArgs.cs

```diff
@@ -16,29 +15,6 @@ namespace Microsoft.VisualStudio.TestPlatform.ObjectModel.Client;
 [DataContract]
 public class DiscoveryCompleteEventArgs : EventArgs
 {
-    /// <summary>
-    /// Constructor for creating event args object
-    /// </summary>
-    /// <param name="totalTests">Total tests which got discovered</param>
-    /// <param name="isAborted">Specifies if discovery has been aborted.</param>
-    /// <param name="fullyDiscoveredSources">List of fully discovered sources</param>
-    /// <param name="partiallyDiscoveredSources">List of partially discovered sources</param>
-    /// <param name="notDiscoveredSources">List of not discovered sources</param>
-    public DiscoveryCompleteEventArgs(
```

Comment:
@cvpoienaru added a new ctor with this signature + the extensions dictionary but this public ctor is not yet released so we can just go with only one overload.